build_all.sh

```diff
@@ -3,21 +3,22 @@ set -eux
 
 target=${target:-"NULL"}
 
-if [[ "$target" == "linux.gnu" || "$target" == "linux.intel" ]]; then
+export MOD_PATH
+
+if [[ "$target" == "linux" || "$target" == "macosx" ]]; then
```

Comment:
do you still need the .intel, .gnu suffices for both linux and macos? If so, use
```
if [[ "$target" == "linux.*" || "$target" == "macosx.*" ]]; then
```
(I think that works, but someone needs to test)